.gitignore

```diff
@@ -1,7 +1,6 @@
 .vscode/
 
 *.pyc
-__pychache__/
```

Comment:
I want this here because of vscode shit.

Reply:
Yeah I also use vscode, but didn't really have a problem with it until you put my attention to it, now I include the whole directory in the .gitignore

Reply:
It still seems to be showing the folder for me tho...

![image](https://user-images.githubusercontent.com/56681655/156362886-fd959c16-695d-4616-b0ed-1889fd384ed8.png)

![image](https://user-images.githubusercontent.com/56681655/156362922-8075de89-0995-4644-ba29-9cf0f0527b9f.png)